install_config/install/docker_registry.adoc

```diff
@@ -1010,19 +1015,31 @@ middleware responsible for interaction with {product-title} and image proxying.
 [source,yaml]
 ----
 middleware:
+  registry:
+    - name: openshift <1>
   repository:
-    - name: openshift
+    - name: openshift <1>
       options:
-        pullthrough: true <1>
-        enforcequota: false <2>
-        projectcachettl: 1m <3>
-----
-<1> Let the registry act as a proxy for remote blobs.
-<2> Prevent blob uploads exceeding size limit defined in targeted project. It
+        acceptschema2: false <2>
+        pullthrough: true <3>
+        enforcequota: false <4>
+        projectcachettl: 1m <5>
+  storage:
+    - name: openshift <1>
+----
+<1> These entries are mandatory.
+<2> Allow to store
+link:https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-2.md#image-manifest-version-2-schema-2[manifest
+schema v2]. If `false`, an attempt to push manifest schema v2 to the registry
+will be refused. As a consequence, client will fallback to pushing
+link:https://github.com/docker/distribution/blob/master/docs/spec/manifest-v2-1.md#image-manifest-version-2-schema-1[manifest
+schema v1].
```

Comment:
Maybe we can add a short explanation of this ?
```
This will help to avoid incompatibilities when using clients docker < 1.10 and docker 1.10 and above at the same time.
```